src/shader_recompiler/frontend/translate/data_share.cpp

```diff
@@ -62,7 +73,25 @@ void Translator::S_BARRIER() {
 }
 
 void Translator::V_READFIRSTLANE_B32(const GcnInst& inst) {
```

Comment:
Keep the assert in V_READFIRSTLANE_B32 as the implementation is a stub that happens to work for RDR only. Also DS_MIN/DS_MAX should be removed